go/go.mod

```diff
@@ -10,6 +10,7 @@ require (
 
 require (
 	github.com/BurntSushi/toml v1.2.1 // indirect
+	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
Do you know why this is added? On my connect branch it is not needed since protobuf is already listed on line 7. If you do `go mod tidy` does this get removed?